sim/verilator.wake

```diff
@@ -13,7 +13,7 @@ tuple VerilatorCompilePlan =
   global Resources:     List String
   global OutputDir:     String
 
-global def defaultVerilatorResources = "verilator/4.008", Nil
+global def defaultVerilatorResources = "verilator/4.024", Nil
```

Comment:
```suggestion
global def defaultVerilatorResources = "verilator/4.028", Nil
```

Actually, I'm going to change this to 4.028, since that's what we're using in a few places internally, and it'd be nice to make them match.